### DISCUSSION TITLE
High level API reference?

### DISCUSSION THREAD

**Discussion by E-Chow** [cvat-ai/cvat#8393] [Q&A]
I don't seem to find the reference page for High level API reference. I found the [documentation](https://docs.cvat.ai/docs/api_sdk/sdk/highlevel-api/) for high level api with some brief intros and examples, but I don't seem to find the reference page like [this](https://docs.cvat.ai/docs/api_sdk/sdk/reference/apis/)

**Comment by ivanvoid** [comment]
+1 have same request. (Maybe start writing it ourselfs coz it open-source idk.)   
here the best thing I found. Just read examples and try to find what you need: [examples for (almost) all high-level APIs](https://github.com/cvat-ai/cvat/issues/7526#issuecomment-1978275592)
here is the fresh link on [python sdk tests](https://github.com/cvat-ai/cvat/blob/develop/tests/python/sdk/test_tasks.py)  
but tbh, it's not helping much
